AutomatticTracks/src/main/java/com/automattic/android/tracks/TracksUser.kt

```diff
@@ -0,0 +1,8 @@
+package com.automattic.android.tracks
+
+data class TracksUser(
+    val userID: String,
+    val email: String,
+    val username: String,
+    val context: Map<String, String>,
```

Comment:
I've decided to move `userContext` into `TracksUser` but I'm thinking about removing it at all. 

- [`WPAndroid` doesn't use it](https://github.com/wordpress-mobile/WordPress-Android/blob/a9eeed79f6d85fd3a3a55157ff9b2932248c00cc/WordPress/src/main/java/org/wordpress/android/util/CrashLogging.kt#L23)
- [`WCAndroid` deosn't use it](https://github.com/woocommerce/woocommerce-android/blob/9e3ee2b151a19d3f2ddf9979147ebaa17fbffee4/WooCommerce/src/main/kotlin/com/woocommerce/android/util/CrashUtils.kt#L68)
- [`simplenote-android` doesn't use it](https://github.com/Automattic/simplenote-android/blob/25c4eccdd4f153ad6a5dda7b4bb7d8de427f337b/Simplenote/src/main/java/com/automattic/simplenote/utils/CrashUtils.java#L94)

Reply:
I agree that it seems the filed isn't being used so we might want to consider removing it. 

One question not related to this PR: Why do we use "TracksUser" for CrashLogging (Tracks vs Sentry). I find having these two tools within one library quite confusing, but using TracksUser just in Sentry and not in Tracks part of the library is even more confusing. I think we might want to consider renaming it and moving it into the "CrashTracking" part of the library. Wdyt?

Reply:
> I find having these two tools within one library quite confusing 

I second to that 👍 

> using TracksUser just in Sentry and not in Tracks part of the library is even more confusing

and to that. Good observation, I'll rename it and move to `crashlogging` package. Thank you!